h/api/search/query.py

```diff
@@ -32,6 +36,7 @@ def build(self, params):
 
         filters = [f(params) for f in self.filters]
         matchers = [m(params) for m in self.matchers]
+        aggregations = dict([(a.key, a()) for a in self.aggregations])
```

Comment:
So's you know, dict comprehensions are a thing: `{a.key: a() for a in self.aggregations}`.